osu.Game/GameModes/Play/Player.cs

```diff
@@ -49,6 +50,8 @@ protected override void Load(BaseGame game)
 
             AudioTrack track = Beatmap.Track;
 
+            background = new BackgroundModeCustom(Beatmap.Background);

```

Comment:
this ain't gonna work as you want it to.. are you on discord so i can explain it to you?